src/bindings/fluid_ladspa.c

```diff
@@ -955,6 +995,13 @@ int fluid_ladspa_check(fluid_ladspa_fx_t *fx, char *err, int err_size)
     const char *str2;
     fluid_ladspa_effect_t *effect;
 
+    fluid_return_val_if_fail(fx != NULL, FLUID_FAILED);
+    fluid_return_val_if_fail(err_size >= 0, FLUID_FAILED);
+    if (err_size > 0 && err == NULL)
+    {
+        return FLUID_FAILED;
+    }
```

Comment:
I support a usecase for allowing `err` to be NULL. But perhaps better consider something like:

```
if (err != NULL)
{
    fluid_return_val_if_fail(size>0, FLUID_FAILED);
}
else;// err msgs not wanted, ignore err_size
```

Better make sure to guard snprintf calls with `err!=NULL`. And consider updating the doc to point out that `err` may be NULL ("if applicable" may also be understood as "err to be filled if a problem detected and unchanged if not").

Reply:
Done.